scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -0,0 +1,294 @@
+version 1.0
+
+# This is a workflow for filtering a joint callset VCF using INFO level annotations (so filtering is at the site level).
+# Note that the input VCFs here may be sharded by genomic position which may be helpful for large cohorts. The script
+# will output the same number of shards that are input.
+# This portion of the filtering pipeline will assign a SCORE INFO field annotation to each site, but does not yet apply
+# the filtering threshold to the final VCF.
+
+workflow JointVcfFiltering {
+	input {
+		Array[File] vcf
+		Array[File] vcf_index
```

Comment:
I agree.  This code was ported directly from the master branch so it's not our code, and will soon be replaced by updated version.  Looking at the latest version of it in master, Sam has refactored it and renamed these variables to input_vcfs and input_vcf_indxs.  In general, looking at his latest file, these pluralization issues have been taken care of which means that when we update to the latest version (after testing this one) this will be fixed automatically.